### PR TITLE
renaming

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/models/_models.py [hunk 1]
```diff
@@ -1904,17 +1904,17 @@ def __init__(self, name, **kwargs):
...
     def __repr__(self):
-        return "NotificationHook(id={}, name={}, description={}, external_link={}, admin_emails={}, " \
+        return "NotificationHook(id={}, name={}, description={}, external_link={}, admins={}, " \
                "hook_type={})".format(
                     self.id,
                     self.name,
                     self.description,
                     self.external_link,
-                    self.admin_emails,
+                    self.emails,
```
Comment (kristapratico) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/models/_models.py:1917]:
```suggestion
                    self.admins,
```

**Review by kristapratico — APPROVED** [review]

**Comment by xiangyan99** [comment]
passed in https://dev.azure.com/azure-sdk/public/_build/results?buildId=980685&view=results

**Pull request merged**